spec/helpers/loss_report_helper_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'Loss Report Helper', js: true do
+  let(:umm_c_version) { '1.15.3' }
+
+  context '#prepare_collections' do
+    context 'when using cmr endpoints' do
+      it 'successfully retrieves and translates the dif collection' do
+        expect(helper.prepare_collections(dif_id, umm_c_version)).to be_truthy
+      end
+      it 'successfully retrieves and translates the iso collection' do
+        expect(helper.prepare_collections(iso_id, umm_c_version)).to be_truthy
+      end
+      it 'successfully retrieves and translates the echo collection' do
+        expect(helper.prepare_collections(echo_id, umm_c_version)).to be_truthy
+      end
+    end
+
+  context '#loss_report_output'
+    context 'when processing a dif collection' do
+      it 'successfully produces a text loss report' do
+        expect(helper.loss_report_output(dif_id).gsub(/\s+/, "")).to eql(dif_text_report.gsub(/\s+/, ""))
+      end
+      it 'successfully produces a json loss report' do
+        expect(helper.loss_report_output(dif_id, hide_items=false, disp='json')).to eql(dif_json_report)
+      end
+    end
+    context 'when processing an echo collection' do
+      it 'successfully produces a text loss report' do
+        expect(helper.loss_report_output(echo_id).gsub(/\s+/, "")).to eql(echo_text_report.gsub(/\s+/, ""))
+      end
+      it 'successfully produces a json loss report' do
+        expect(helper.loss_report_output(echo_id, hide_items=false, disp='json')).to eql(echo_json_report)
+      end
+    end
+    context 'when processing an iso collection' do
+      it 'successfully produces a text loss report' do
+        expect(helper.loss_report_output(iso_id).gsub(/\s+/, "")).to eql(iso_text_report.gsub(/\s+/, ""))
+      end
+      it 'successfully produces a json loss report' do
```

Comment:
I think I would prefer this to have a larger fingerprint for the record.

Reply:
Yes, will do.